src/coreclr/nativeaot/Runtime/arm64/ExceptionHandling.S

```diff
@@ -301,6 +301,7 @@ LOCAL_LABEL(TailCallWasHijacked):
         mov         lr, x1
         str         lr, [sp, #(rsp_offsetof_Context + OFFSETOF__PAL_LIMITED_CONTEXT__LR)]
         str         lr, [sp, #(rsp_offsetof_Context + OFFSETOF__PAL_LIMITED_CONTEXT__IP)]
+        xpaclri
```

Comment:
Are the tests able to hit this path?

It does not look right that we are clearing the signature for `lr` on `TailCallWasHijacked` path, but not on the regular path.

Reply:
I'll take a look again.